jdk 15 failed on Mac

jdk15 on Mac got:

```
Creating support/modules_cmds/openj9.dtfj/jextract from 1 file(s)
/Users/runner/work/runaqaTest/runaqaTest/openj9-openjdk-jdk15/src/java.desktop/macosx/native/libawt_lwawt/awt/CSystemColors.m:129:9: error: converting the result of '?:' with integer constants to a boolean always evaluates to 'true' [-Werror,-Wtautological-constant-compare]
    if (colorIndex < (useAppleColor) ? sun_lwawt_macosx_LWCToolkit_NUM_APPLE_COLORS : java_awt_SystemColor_NUM_COLORS) {
        ^
1 error generated.
make[3]: *** [/Users/runner/work/runaqaTest/runaqaTest/openj9-openjdk-jdk15/build/macosx-x86_64-server-release/support/native/java.desktop/libawt_lwawt/CSystemColors.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [java.desktop-libs] Error 2
```

https://github.com/sophia-guo/runaqaTest/runs/1419354025?check_suite_focus=true

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -40,6 +40,8 @@ jobs:
             version: 8 
           - os: windows-2016
             version: 11 #https://github.com/eclipse/build-openj9/issues/2
+          - os: macos-10.15
+            version: 15 #https://github.com/ibmruntimes/openj9-openjdk-jdk15/issues/47#issuecomment-736017181
     steps:
     - uses: actions/checkout@v1
     - uses: eclipse/build-openj9@v1

